ruby/ql/test/library-tests/frameworks/app/models/user.rb

```diff
@@ -0,0 +1,2 @@
+class User < ApplicationRecord
+end
```

Comment:
It would be good to have some calls in this class to test we identify them as well, since that's quite common. E.g.

```rb
class User
  def make_admin!
    update!(is_admin: true)
  end
end
```